manim/mobject/svg/text_mobject.py

```diff
@@ -297,7 +297,7 @@ def text2svg(self):
         file_name = os.path.join(dir_name, hash_name) + ".svg"
         if os.path.exists(file_name):
             return file_name
-        surface = cairo.SVGSurface(file_name, 600, 400)
+        surface = cairo.SVGSurface(file_name, len(self.text) * size, 400)
```

Comment:
I hate hard-coding any part of this. Why was it set to 600, 400??? Oh man.

Question: can we set the surface size to cover the whole manim canvas? Would that slow things down considerably?

Reply:
While we could do that, that might not be enough: The example I have above would not fit on the entire canvas, but after scaling the text, it does.

Regarding render speed: I am not sure in how far the size of the canvas actually influences the required time; in the experiments I did I couldn't really observe an influence (although my change increases the canvas size on average by a factor of at least 2).